staging/src/k8s.io/apimachinery/pkg/runtime/caching_object.go

```diff
@@ -0,0 +1,162 @@
+/*
```

Comment:
I'd definitely prefer the contents of this file to go in another package, preferably not in api machinery at all.